spawn error on startup (Windows 7)

I totally understand if Windows isn't really supported yet, but thought I'd give you a heads up that I'm hitting an issue upon startup.

**With this run script:**
```js
"scripts": {
    "dev": "webpack-dashboard -- webpack-dev-server --config ./webpack.dev.js"
}
```

**And this in my webpack plugins:**
```js
plugins: [
    new DashboardPlugin()
]
```

**I get the following error:**
```bash
> webpack-dashboard -- webpack-dev-server --config ./webpack.dev.config.js

Error: spawn webpack-dev-server ENOENT
    at exports._errnoException (util.js:1007:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:182:32)
    at onErrorNT (internal/child_process.js:348:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
    at Function.Module.runMain (module.js:577:11)
    at startup (node.js:160:18)
    at node.js:456:3

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "dev:app"
npm ERR! node v6.2.1
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
```

--- a/bin/webpack-dashboard.js
+++ b/bin/webpack-dashboard.js
@@ -25,6 +25,8 @@ if (!program.args.length) {
 var command = program.args[0];
 var args = program.args.slice(1);
 
+args.push("--color");
+
 var child = spawn(command, args, {
   stdio: [null, null, null, "ipc"],
   detached: true

--- a/bin/webpack-dashboard.js
+++ b/bin/webpack-dashboard.js
@@ -2,7 +2,7 @@
 "use strict";
 
 var commander = require("commander");
-var spawn = require("child_process").spawn;
+var spawn = require("cross-spawn");
 var Dashboard = require("../dashboard/index.js");
 var net = require("net");
 var JsonSocket = require("json-socket");
@@ -26,7 +26,8 @@ var command = program.args[0];
 var args = program.args.slice(1);
 
 var child = spawn(command, args, {
-  stdio: [null, null, null, 'ipc']
+  stdio: [null, null, null, 'ipc'],
+  detached: true
 });
 
 var dashboard = new Dashboard({
@@ -63,5 +64,5 @@ child.stderr.on('data', function (data) {
 });
 
 process.on('exit', function () {
-  child.kill();
+  process.kill(-child.pid);
 });

--- a/bin/webpack-dashboard.js
+++ b/bin/webpack-dashboard.js
@@ -11,9 +11,9 @@ var program = new commander.Command("webpack-dashboard");
 
 var pkg = require("../package.json");
 program.version(pkg.version);
-program.option('-c, --color [color]', 'Dashboard color');
-program.option('-m, --minimal', 'Minimal mode');
-program.option('-p, --port [port]', 'Socket listener port');
+program.option("-c, --color [color]", "Dashboard color");
+program.option("-m, --minimal", "Minimal mode");
+program.option("-p, --port [port]", "Socket listener port");
 program.usage("[options] -- [script] [arguments]");
 program.parse(process.argv);
 
@@ -26,7 +26,7 @@ var command = program.args[0];
 var args = program.args.slice(1);
 
 var child = spawn(command, args, {
-  stdio: [null, null, null, 'ipc'],
+  stdio: [null, null, null, "ipc"],
   detached: true
 });
 
@@ -38,31 +38,31 @@ var dashboard = new Dashboard({
 var port = program.port || 9838;
 var server = net.createServer();
 server.listen(port);
-server.on('connection', function(socket) {
+server.on("connection", function(socket) {
   socket = new JsonSocket(socket);
-  socket.on('message', function(message) {
+  socket.on("message", function(message) {
     dashboard.setData(message);
   });
 });
 
-server.on('error', function(err) {
+server.on("error", function(err) {
   console.log(err);
 });
 
-child.stdout.on('data', function (data) {
+child.stdout.on("data", function (data) {
   dashboard.setData([{
-    type: 'log',
+    type: "log",
     value: data.toString("utf8")
   }]);
 });
 
-child.stderr.on('data', function (data) {
+child.stderr.on("data", function (data) {
   dashboard.setData([{
-    type: 'error',
+    type: "error",
     value: data.toString("utf8")
   }]);
 });
 
-process.on('exit', function () {
+process.on("exit", function () {
   process.kill(-child.pid);
 });

--- a/dashboard/index.js
+++ b/dashboard/index.js
@@ -1,11 +1,11 @@
 /* eslint-disable */
-'use strict';
+"use strict";
 
-var blessed = require('blessed');
+var blessed = require("blessed");
 
-var formatOutput = require('../utils/format-output.js');
-var formatModules = require('../utils/format-modules.js');
-var formatAssets = require('../utils/format-assets.js');
+var formatOutput = require("../utils/format-output.js");
+var formatModules = require("../utils/format-modules.js");
+var formatAssets = require("../utils/format-assets.js");
 
 function Dashboard(options) {
   this.color = options && options.color || "green";
@@ -14,7 +14,7 @@ function Dashboard(options) {
 
   this.screen = blessed.screen({
     smartCSR: true,
-    title: 'webpack-dashboard',
+    title: "webpack-dashboard",
     dockBorders: false,
     fullUnicode: true,
     autoPadding: true
@@ -25,7 +25,7 @@ function Dashboard(options) {
   !this.minimal && this.layoutModules.call(this);
   !this.minimal && this.layoutAssets.call(this);
 
-  this.screen.key(['escape', 'q', 'C-c'], function() {
+  this.screen.key(["escape", "q", "C-c"], function() {
     process.exit(0);
   });
 
@@ -37,36 +37,36 @@ Dashboard.prototype.setData = function(dataArr) {
 
   dataArr.forEach(function(data) {
     switch (data.type) {
-      case 'progress': {
+      case "progress": {
         var percent = parseInt(data.value * 100);
         if (self.minimal) {
-          percent && self.progress.setContent(percent.toString() + '%');
+          percent && self.progress.setContent(percent.toString() + "%");
         } else {
           self.progressbar.setProgress(percent);
         }
         break;
       }
-      case 'operations': {
+      case "operations": {
         self.operations.setContent(data.value);
         break;
       }
-      case 'status': {
+      case "status": {
         var content;
 
         switch(data.value) {
-          case 'Success':
-            content = '{green-fg}{bold}' + data.value + '{/}';
+          case "Success":
+            content = "{green-fg}{bold}" + data.value + "{/}";
             break;
-          case 'Failed':
-            content = '{red-fg}{bold}' + data.value + '{/}';
+          case "Failed":
+            content = "{red-fg}{bold}" + data.value + "{/}";
             break;
           default:
-            content = '{white-fg}{bold}' + data.value + '{/}';
+            content = "{bold}" + data.value + "{/}";
         }
         self.status.setContent(content);
         break;
       }
-      case 'stats': {
+      case "stats": {
         var stats = {
           hasErrors: function() {
             return data.value.errors;
@@ -79,23 +79,23 @@ Dashboard.prototype.setData = function(dataArr) {
           }
         };
         if (stats.hasErrors()) {
-          self.status.setContent('{red-fg}{bold}Failed{/}');
+          self.status.setContent("{red-fg}{bold}Failed{/}");
         }
         self.logText.log(formatOutput(stats));
         !self.minimal && self.moduleTable.setData(formatModules(stats));
         !self.minimal && self.assetTable.setData(formatAssets(stats));
         break;
       }
-      case 'log': {
+      case "log": {
         self.logText.log(data.value);
         break;
       }
-      case 'error': {
+      case "error": {
         self.logText.log("{red-fg}" + data.value + "{/}");
         break;
       }
-      case 'clear': {
-        self.logText.setContent('');
+      case "clear": {
+        self.logText.setContent("");
         break;
       }
     }
@@ -106,17 +106,17 @@ Dashboard.prototype.setData = function(dataArr) {
 
 Dashboard.prototype.layoutLog = function() {
   this.log = blessed.box({
-    label: 'Log',
+    label: "Log",
     padding: 1,
-    width: this.minimal ? "100%" : '75%',
-    height: this.minimal ? "70%" : '42%',
-    left: '0%',
-    top: '0%',
+    width: this.minimal ? "100%" : "75%",
+    height: this.minimal ? "70%" : "42%",
+    left: "0%",
+    top: "0%",
     border: {
-      type: 'line',
+      type: "line",
     },
     style: {
-      fg: 'white',
+      fg: -1,
       border: {
         fg: this.color,
       },
@@ -131,7 +131,7 @@ Dashboard.prototype.layoutLog = function() {
     input: true,
     alwaysScroll: true,
     scrollbar: {
-      ch: ' ',
+      ch: " ",
       inverse: true
     },
     keys: true,
@@ -144,18 +144,18 @@ Dashboard.prototype.layoutLog = function() {
 
 Dashboard.prototype.layoutModules = function() {
   this.modules = blessed.box({
-    label: 'Modules',
+    label: "Modules",
     tags: true,
     padding: 1,
-    width: '50%',
-    height: '58%',
-    left: '0%',
-    top: '42%',
+    width: "50%",
+    height: "58%",
+    left: "0%",
+    top: "42%",
     border: {
-      type: 'line',
+      type: "line",
     },
     style: {
-      fg: 'white',
+      fg: -1,
       border: {
         fg: this.color,
       },
@@ -172,13 +172,13 @@ Dashboard.prototype.layoutModules = function() {
     scrollable: true,
     alwaysScroll: true,
     scrollbar: {
-      ch: ' ',
+      ch: " ",
       inverse: true
     },
     keys: true,
     vi: true,
     mouse: true,
-    data: [['Name', 'Size', 'Percentage']]
+    data: [["Name", "Size", "Percentage"]]
   });
 
   this.screen.append(this.modules);
@@ -186,18 +186,18 @@ Dashboard.prototype.layoutModules = function() {
 
 Dashboard.prototype.layoutAssets = function() {
   this.assets = blessed.box({
-    label: 'Assets',
+    label: "Assets",
     tags: true,
     padding: 1,
-    width: '50%',
-    height: '58%',
-    left: '50%',
-    top: '42%',
+    width: "50%",
+    height: "58%",
+    left: "50%",
+    top: "42%",
     border: {
-      type: 'line',
+      type: "line",
     },
     style: {
-      fg: 'white',
+      fg: -1,
       border: {
         fg: this.color,
       },
@@ -213,13 +213,13 @@ Dashboard.prototype.layoutAssets = function() {
     scrollable: true,
     alwaysScroll: true,
     scrollbar: {
-      ch: ' ',
+      ch: " ",
       inverse: true
     },
     keys: true,
     vi: true,
     mouse: true,
-    data: [['Name', 'Size']]
+    data: [["Name", "Size"]]
   });
 
   this.screen.append(this.assets);
@@ -237,19 +237,19 @@ Dashboard.prototype.layoutStatus = function() {
 
   this.status = blessed.box({
     parent: this.wrapper,
-    label: 'Status',
+    label: "Status",
     tags: true,
     padding: {
       left: 1,
     },
-    width: this.minimal ? "34%" : '100%',
-    height: this.minimal ? "100%" : '34%',
+    width: this.minimal ? "34%" : "100%",
+    height: this.minimal ? "100%" : "34%",
     valign: "middle",
     border: {
-      type: 'line',
+      type: "line",
     },
     style: {
-      fg: 'white',
+      fg: -1,
       border: {
         fg: this.color,
       },
@@ -258,19 +258,19 @@ Dashboard.prototype.layoutStatus = function() {
 
   this.operations = blessed.box({
     parent: this.wrapper,
-    label: 'Operation',
+    label: "Operation",
     tags: true,
     padding: {
       left: 1,
     },
-    width: this.minimal ? "34%" : '100%',
-    height: this.minimal ? "100%" : '34%',
+    width: this.minimal ? "34%" : "100%",
+    height: this.minimal ? "100%" : "34%",
     valign: "middle",
     border: {
-      type: 'line',
+      type: "line",
     },
     style: {
-      fg: 'white',
+      fg: -1,
       border: {
         fg: this.color,
       },
@@ -279,19 +279,19 @@ Dashboard.prototype.layoutStatus = function() {
 
   this.progress = blessed.box({
     parent: this.wrapper,
-    label: 'Progress',
+    label: "Progress",
     tags: true,
     padding: this.minimal ? {
       left: 1,
     } : 1,
-    width: this.minimal ? "33%" : '100%',
-    height: this.minimal ? "100%" : '34%',
+    width: this.minimal ? "33%" : "100%",
+    height: this.minimal ? "100%" : "34%",
     valign: "middle",
     border: {
-      type: 'line',
+      type: "line",
     },
     style: {
-      fg: 'white',
+      fg: -1,
       border: {
         fg: this.color,
       },

--- a/package.json
+++ b/package.json
@@ -29,6 +29,7 @@
   "dependencies": {
     "blessed": "^0.1.81",
     "commander": "^2.9.0",
+    "cross-spawn": "^4.0.0",
     "filesize": "^3.3.0",
     "json-socket": "^0.1.2",
     "webpack": "^1.13.1"

--- a/plugin/index.js
+++ b/plugin/index.js
@@ -23,10 +23,10 @@ DashboardPlugin.prototype.apply = function(compiler) {
   if (!handler) {
     handler = noop;
     var port = this.port;
-    var host = '127.0.0.1';
+    var host = "127.0.0.1";
     var socket = new JsonSocket(new net.Socket());
     socket.connect(port, host);
-    socket.on('connect', function() {
+    socket.on("connect", function() {
       handler = socket.sendMessage.bind(socket);
     });
   }

